spec.html

```diff
@@ -30984,16 +30984,19 @@
           1. If _f_ is not *undefined*, then
             1. If IsCallable(_f_) is *false*, throw a *TypeError* exception.
           1. If _thisArg_ was supplied, let _t_ be _thisArg_; else let _t_ be *undefined*.
-          1. Return TypedArrayFrom(_C_, _source_, _f_, _t_).
+          1. Return ? TypedArrayFrom(*undefined*, _C_, _source_, _f_, _t_).
         </emu-alg>
 
         <!-- es6num="22.2.2.1.1" -->
         <emu-clause id="sec-typedarrayfrom" aoid="TypedArrayFrom">
-          <h1>Runtime Semantics: TypedArrayFrom( _constructor_, _items_, _mapfn_, _thisArg_ )</h1>
-          <p>When the TypedArrayFrom abstract operation is called with arguments _constructor_, _items_, _mapfn_, and _thisArg_, the following steps are taken:</p>
+          <h1>Runtime Semantics: TypedArrayFrom( _O_, _constructor_, _items_, _mapfn_, _thisArg_ )</h1>
```

Comment:
The signature of this AO is a bit strange now, since you either pass the first argument or the second.  Is there some refactoring that needs to be done to clean this up?